sympy/combinatorics/coset_table.py

```diff
@@ -816,6 +816,17 @@ def conjugates(self, R):
             R_set.difference_update(r)
         return R_c_list
 
+    def coset_representative(self, coset):
+        '''
+        Compute the coset representative of a given coset.
+        To-do: Add examples
+        '''
+        for alpha, x in product(self.omega, self.A):
+            if self.table[alpha][self.A_dict[x]] == coset:
+                if alpha == 0:
+                    return x
+                return self.coset_representative(alpha)*x
+
```

Comment:
You are not using this anywhere else. I suppose it could be good to have this sort of a function but it can be more efficient: for example, you could start in row `coset`, find the first `x` s.t. `gamma < coset` where `gamma = self.table[coset][self.A_dict[x]]` and return `coset_representative(gamma)*x**-1`